README.md

```diff
@@ -31,6 +32,19 @@ What's included in this template:
  * Pytest example(s).
  * Githooks.
 
+# Quickstart Guide
+
+Add here, streamlined instructions on how to get running and using the code as swiftly as possible. This shouldn't
```

Comment:
Using both verbs "running" and "using" feels a bit clunky. Perhaps change to:

Add here, streamlined instructions on how to get **the code running** as swiftly as possible.